cli/fathom_web/test/test_extract.py

```diff
@@ -0,0 +1,32 @@
+from ..commands.extract import BASE64_DATA_PATTERN
+
+
+def test_common_example():
+    """Confirm we handle to well behaving case"""
+    mime_type = 'image/png'
+    base64_string = 'aorienstar/tar/ararnsoine98daQAAAIST+++/rstienf='
+    test_string = f'data:{mime_type};base64,{base64_string}'
+    matches = list(BASE64_DATA_PATTERN.finditer(test_string))
+    assert len(matches) == 1
+    assert matches[0].group('mime') == mime_type
+    assert matches[0].group('string') == base64_string
+
+
+def test_empty_string():
+    """Some base64 strings are actually empty"""
+    test_string = 'data:;base64,'
+    matches = list(BASE64_DATA_PATTERN.finditer(test_string))
```

Comment:
This line would probably be a good thing to factor out into a test-helper procedure. It's used in every test, it's got a few different pieces of knowledge in it, and I already requested one tweak to it.

Reply:
That's reasonable. I'm on it.